var/spack/repos/builtin/packages/pmt/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pmt(CMakePackage):
+    """A multi-runtime implementation of a distributed merge tree
+    segmentation algorithm. The implementation relies on the framework
+    BabelFlow, which allows to execute the algorithm on different runtime
+    systems."""
+
+    homepage = "https://bitbucket.org/cedmav/parallelmergetree"
+    url      = "https://bitbucket.org/cedmav/parallelmergetree/get/ascent.zip"
+
+    maintainers = ['spetruzza']
+
+    version('develop',
+            git='https://bitbucket.org/cedmav/parallelmergetree.git',
+            branch='ascent',
+            commit='6774ed74fd13b9747ac792978a676ce6e8b05cab',
```

Comment:
Do branches and commits work together? I've only ever seen one or the other in use.

Reply:
I checked this out, it turns out they are both the same -- so at least there is no mystery about which is right. 
I am going to stick with the commit.

Reply:
fixed this